spec/features/feedback_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.feature 'Feedback form', js: true do
+
+  before do
+    stub_request(:post, "#{ENV.fetch('ET_API_URL', 'http://api.et.127.0.0.1.nip.io:3100/api')}/v2/feedback/build_feedback")
```

Comment:
When you think about this code, you are potentially using the ET_API_URL environment variable in the stub_request in line 6, but not in line 16 where you say 
```
expect(a_request(:post, "http://api.et.127.0.0.1.nip.io:3100/api/v2/feedback/build_feedback").
```
this means that if you had this environment variable set to something else your test would fail 

What you can do as an alternative strategy is rather than relying on an external environment variable being set, you set it YOURSELF in the spec (making sure you put it back afterwards so you dont affect other tests) to some hard coded value (can be anything - even http://www.google.com if you want !) - then the test will be more robust.

To do this easily, there is a gem called 'climate control' (https://github.com/thoughtbot/climate_control) - we dont use it in this project, but consider it for next time (its pointless changing this one as other tests also do the same)